client/protocols/collaborate.py

```diff
@@ -73,9 +73,8 @@ def run(self, messages):
             log.warning("Exception while waiting", exc_info=True)
 
     def start_firebase(self, messages):
-        access_token = auth.authenticate(self.assignment, force=False)
```

Comment:
Sidenote: The collab server will need to have some changes to handle having access tokens coming directly from OK (instead of Google)